tests/acceptance/25_cf-execd/mail_include_filter.cf

```diff
@@ -0,0 +1,38 @@
+# Tests that include mail filters, causes only those lines to be mailed by
+# cf-execd.
```

Comment:
Maybe " Tests that include mail filter causes only matched lines are mailed by cf-execd."?